commit ded6bd330906152f5f233f5859740954a78dea49
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Fri Aug 26 23:55:31 2022 +0200

    disable setupcon warning
    
    at least have it be quiet while we don't have a solution yet

diff --git a/hooks/keymap b/hooks/keymap
index e2d4e55..ad49c0a 100755
--- a/hooks/keymap
+++ b/hooks/keymap
@@ -21,7 +21,6 @@ if [ "$KEYMAP" != "y" ] && [ "$KEYMAP" != "Y" ]; then
 fi
 
 if [ ! -x /usr/bin/setupcon ]; then
-	echo "setupcon is missing. Please install the 'console-setup' package."
 	exit 0
 fi
 
